python/lsst/alert/packet/updateSchema.py

```diff
@@ -0,0 +1,144 @@
+# This file is part of alert_packet.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import argparse
+import os
+import fastavro
+import yaml
+import json
+from lsst.alert.packet import get_schema_root, SchemaRegistry
+
+__all__ = ['update_schema']
+
+
+def update_docs(schema, apdb):
+    for index, schema_field in enumerate(schema['fields']):
+        if apdb:
+            for apdb_field in apdb['columns']:
+                if schema_field['name'] == apdb_field['name']:
+                    if 'doc' not in schema_field:
+                        schema_field['doc'] = ""
+                    if not schema_field['doc'] == (apdb_field['description']):
+                        schema_field['doc'] = apdb_field['description']
+    return schema
+
+
+def match_table(name, apdb):
+
+    for table in apdb['tables']:
+        if name.lower() == table['name'].lower():
+            return table
+
+
+def write_schema(schema, schema_name, version, path, update_version=False):
+
+    if update_version is True:
+
+        version = version.split('.')
+        schema['namespace'] = schema['namespace'][0:-1] + str(int(version[1]) + 1)
+        updated_schema_loc = path[0:-1] + str(int(version[1]) + 1)
+        updated_schema_name = schema['namespace'] + '.' + schema['name'] + '.avsc'
+
+        if not os.path.exists(updated_schema_loc):
+            os.mkdir(updated_schema_loc)
+        with open(updated_schema_loc + '/' + updated_schema_name, "w") as f:
+            json.dump(schema, f, indent=2)
+
+    else:
+        with open(path + '/' + schema_name, 'w') as f:
+            json.dump(schema, f, indent=2)
+
+
+def add_namespace(schema):
+
+    schema_names = schema['name'].split(".")
+    schema['name'] = schema_names[2]
+    schema['namespace'] = schema_names[0] + "." + schema_names[1]
+    schema = dict(sorted(schema.items(), reverse=True))
+
+    return schema
+
+
+def update_schema(apdb_filepath, update_version=False):
+
+    """Compare an avro schemas  docstrings with the apdb.yaml file.
```

Comment:
Again, I think what we want here is not to compare and only add the docstring, but to generate the whole avsc file from the apdb.yaml.